bin/run_functions.js

```diff
@@ -242,7 +256,7 @@ function executeProject(filename, project, options) {
     async.series([
             // First create a zip file, if needed.
             //
-        function (callback) {
+            function (callback) {
```

Comment:
Can we sync indentation to the rest of the functions?

Reply:
Absolutely!  Done and done.